src/rgw/rgw_lc.cc

```diff
@@ -828,15 +828,38 @@ int RGWLC::handle_multipart_expiration(rgw::sal::Bucket* target,
   params.ns = RGW_OBJ_NS_MULTIPART;
   params.access_list_filter = &mp_filter;
 
+  auto event_type = rgw::notify::ObjectExpirationAbortMPU;
+  std::string version_id;
```

Comment:
i dont see `version_id` being set anywhere. could just be an empty string passed to `publish_commit()`

Reply:
(this matches the other usage in rgw_lc.cc, so don't want to change it)